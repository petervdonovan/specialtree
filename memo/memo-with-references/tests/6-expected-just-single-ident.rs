use memo_with_references::memo;

#[memo(cache other)]
fn memome(cache: &'a str) {}

fn main() {}
