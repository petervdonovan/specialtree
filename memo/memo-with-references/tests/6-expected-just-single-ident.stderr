error: expected just a single ident
 --> tests/6-expected-just-single-ident.rs:3:8
  |
3 | #[memo(cache other)]
  |        ^^^^^^^^^^^
